### PR TITLE
Tried to make it look better than your mom.

### PR DESCRIPTION
**Pull request by GelaniNijraj**
I don't use chrome so I'm not really sure if I've broken anything. Sorry about that.

### CODE REVIEW

**Comment by GelaniNijraj** [comment]
Here's a pic.
screenshot

**Comment by tcmcgee** [comment]
Here's a picture in chrome!

sample2

File: RedditCrap/app/css/popup.css [hunk 1]
```diff
@@ -1,21 +1,104 @@
...
+    border-right: none;
+    width: 195px;
+    height: 100%;
+    box-sizing: border-box;
+    font-size: 14px;
+    border-radius: 3px 0 0 3px;
+    padding: 0 3px;
+    transition: 0.2s;
+}
+input[type="text"]:hover{
```
Comment (Vannevelj) [inline RedditCrap/app/css/popup.css]: Can we simplify this to `border-color: #B5B5B5;` (and same for the `:focus`) event?

File: RedditCrap/app/css/popup.css [hunk 2]
```diff
@@ -1,21 +1,104 @@
...
+    width: 100%;
+    background: #FFFFFF;
+}
+
+option{
+    height: 25px;
+    padding-top: 10px;
+}
+
+#reset{
```
Comment (Vannevelj) [inline RedditCrap/app/css/popup.css]: Perhaps add a `margin-bottom: 5px` to make it feel less clamped against that bottom border

**Comment by GelaniNijraj** [comment]
Like this?

**Comment by GelaniNijraj** [comment]
Okay, I think I got it now.

**Comment by GelaniNijraj** [comment]
Undid that by mistake..

**Comment by Vannevelj** [comment]
Perfect! I appreciate the patience man -- you made it look a lot better.

**Comment by GelaniNijraj** [comment]
Thanks.. :smile:

**Pull request merged**